clusterloader2/testing/load/config.yaml

```diff
@@ -15,22 +16,39 @@
 {{$bigRCsPerNamespace := DivideInt $bigRCs $namespaces}}
 {{$mediumRCsPerNamespace := DivideInt $mediumRCs $namespaces}}
 {{$smallRCsPerNamespace := DivideInt $smallRCs $namespaces}}
-{{$totalPodsRounded := AddInt (MultiplyInt $bigRCs $BIG_GROUP_SIZE) (MultiplyInt $mediumRCs $MEDIUM_GROUP_SIZE) (MultiplyInt $smallRCs $SMALL_GROUP_SIZE)}}
+{{$totalPods := AddInt (MultiplyInt $bigRCs $BIG_GROUP_SIZE) (MultiplyInt $mediumRCs $MEDIUM_GROUP_SIZE) (MultiplyInt $smallRCs $SMALL_GROUP_SIZE)}}
 # saturationQPS - qps for RC operations should be an equivalent to 10 [pods/s].
-{{$saturationQPS := DivideFloat (AddInt $bigRCs $mediumRCs $smallRCs) (DivideFloat $totalPodsRounded 10)}}
+{{$bigSaturationQPS := DivideFloat $bigRCs (DivideFloat $totalPods 10)}}
+{{$mediumSaturationQPS := DivideFloat $mediumRCs (DivideFloat $totalPods 10)}}
+{{$smallSaturationQPS := DivideFloat $smallRCs (DivideFloat $totalPods 10)}}
 #Test
 name: load
 automanagedNamespaces: {{$namespaces}}
 tuningSets:
-- name: AvgSaturationQPS
+- name: SvcSaturationQPS
+  qpsLoad:
+    qps: {{$SVC_SATURATION_QPS}}
+- name: BigAvgSaturationQPS
   randomizedLoad:
-    averageQps: {{$saturationQPS}}
-- name: AvgScalingQPS
+    averageQps: {{$bigSaturationQPS}}
+- name: MediumAvgSaturationQPS
+  randomizedLoad:
+    averageQps: {{$mediumSaturationQPS}}
+- name: SmallAvgSaturationQPS
+  randomizedLoad:
+    averageQps: {{$smallSaturationQPS}}
+- name: BigAvgScalingQPS
   randomizedLoad:
     # The expected number of created/deleted pods is totalPods/4 when scaling,
     # as each RC changes its size from X to a uniform random value in [X/2, 3X/2].
     # To match 10 [pods/s] requirement, we need to multiply saturationQPS by 4.
-    averageQps: {{MultiplyFloat $saturationQPS 4}}
+    averageQps: {{MultiplyFloat $bigSaturationQPS 4}}
+- name: MediumAvgScalingQPS
+  randomizedLoad:
+    averageQps: {{MultiplyFloat $mediumSaturationQPS 4}}
+- name: SmallAvgScalingQPS
+  randomizedLoad:
+    averageQps: {{MultiplyFloat $smallSaturationQPS 4}}
 steps:
 - measurements:
   - Identifier: PhaseTimer
```

Comment:
Can you now get rid of those timers and instead provide names for steps (if we believe those times are useful).
Maybe even a separate PR, to already simplify configs asap.

Reply:
#263 - Removing timers from test configs.

Reply:
Changes applied.